current-drug-targets.csl

```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" page-range-format="minimal" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Current Drug Targets</title>
+    <title-short>Curr Drug Tar</title-short>
+    <id>http://www.zotero.org/styles/current-drug-targets</id>
+    <link href="http://www.zotero.org/styles/current-drug-targets" rel="self"/>
+    <link href="http://www.zotero.org/styles/elsevier-vancouver" rel="template"/>
+    <link href="http://www.elsevier.com/journals/journal-of-crohns-and-colitis-supplements/1873-9954/guide-for-authors" rel="documentation"/>
```

Comment:
@AKREMA007, could you update the "template" link (should point to "journal-of-crohns-and-colitis-supplements"), "documentation" link ("http://benthamscience.com/journals/current-drug-targets/author-guidelines/"), ISSNs (1873-5592 (Online), 1389-4501 (Print)), and summary?